org.eclipse.gef/src/org/eclipse/gef/internal/ui/rulers/GuideEditPart.java

```diff
@@ -112,7 +112,7 @@ protected IFigure createFigure() {
 		return new GuideFigure(isHorizontal());
 	}
 
-	protected GuideLineFigure createGuideLineFigure() {
+	private static GuideLineFigure createGuideLineFigure() {
 		return new GuideLineFigure();
```

Comment:
Same as in the DragGuidePolicy. I would just inline this method.

Reply:
done as suggested.